README.md

```diff
@@ -61,6 +61,7 @@ Environmental variables to configure production release:
 * `PUSH_HTTPS_PORT` - The port of the MongoosePush REST endpoint. Please not that docker exposes only `8443` port, so changing this setting is not recommended
 * `PUSH_HTTPS_KEYFILE` - Path to PEM keyfile used for REST endpoint
 * `PUSH_HTTPS_CERTFILE` - Path to PEM certfile used for REST endpoint
+* `PUSH_HTTPS_ACCEPTORS` - Number of TCP acceptors to start
```

Comment:
It's not, I forgot about that. I can move it to separate PR if you want to. Theres no good place where it fits.